awx/api/serializers.py

```diff
@@ -2778,6 +2778,7 @@ class Meta:
             'inventory',
             'project',
             'playbook',
+            'fqcn_role',
```

Comment:
Not sure how I feel about the name here... "role" is mostly already defined for the rbac system.